docs/guide/content_rendering/urls_and_routes.md

```diff
@@ -0,0 +1,52 @@
+# URLs and routes
+
+To link to a Location or Content item, use the [`ez_path()`](twig_function_reference/url_twig_functions.md#ez_path) Twig function.
```

Comment:
I think we must assume that it's clear to the audience why they'd want to add links, at least in developer doc. It could be clarified in user doc, but not here.